From 7237f16554ca2e0c98bf324e9a1ca9c8125b0a18 Mon Sep 17 00:00:00 2001
From: Duncan Macleod <macleoddm@cardiff.ac.uk>
Date: Fri, 5 Dec 2025 13:27:12 +0000
Subject: [PATCH] Use platform-independent script entry point

Rather than script
---
 pyproject.toml | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index f3a2e62..201d331 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -31,12 +31,14 @@ dependencies = [
     "requests",
 ]
 
+[project.scripts]
+gpstime = "gpstime.__main__:main"
+
 [project.urls]
 Homepage = "https://git.ligo.org/cds/software/gpstime"
 
 [tool.setuptools]
 packages = ["gpstime"]
-script-files = ["bin/gpstime"]
 
 [tool.setuptools_scm]
 write_to = "gpstime/__version__.py"
-- 
2.39.5

