diff --git a/setup.py b/setup.py
index 501c10b..1be3903 100644
--- a/setup.py
+++ b/setup.py
@@ -48,12 +48,9 @@ setup(
         'requests',
     ],
 
-    # entry_points={
-    #     'console_scripts': [
-    #         'gpstime = gpstime.__main__:main',
-    #     ],
-    # },
-    scripts=[
-        'bin/gpstime',
-    ],
+    entry_points={
+        'console_scripts': [
+            'gpstime = gpstime.__main__:main',
+        ],
+    },
 )
